### DISCUSSION TITLE
[Feature request] 控制器文件夹分组路由

### DISCUSSION THREAD

**Discussion by whlsxl** [midwayjs/midway#3746] [Ideas]
### The feature, motivation（功能、动机）

看到https://github.com/midwayjs/midway/issues/219 讨论分组路由的问题，最后给出的方案是使用[动态添加 Web 控制器](https://midwayjs.org/docs/router_table#%E6%B7%BB%E5%8A%A0%E8%B7%AF%E7%94%B1)。虽然可以为同一目录下的控制器挨个添加路由，但是这样就放弃使用@Controller，整体代码风格也变了。

能否添加一种方式，比如在同目录下的`index.ts`文件，添加`prefix`配置

export default {
  routerPrefix: 'admin',
}

这样保证了装饰器风格一致，也没有破坏`IoC`，路由也没有依赖文件夹而是依赖配置。

**Comment by czy88840616** [comment]
文件路由不会考虑自建，可能直接用 nextjs 的了
